app/test_main.py

```diff
@@ -1 +1,59 @@
-# write your code here
+from app.main import can_access_google_page
+from unittest import mock
+import pytest
+
+
+@pytest.fixture()
+def mocked_valid_url():
+    with mock.patch("app.main.valid_google_url") as mock_valid_url:
+        yield mock_valid_url
+
+
+@pytest.fixture()
+def mocked_has_internet_connection():
+    with mock.patch("app.main.has_internet_connection") as mock_has_internet:
+        yield mock_has_internet
+
+
+def test_function_correct_work(mocked_valid_url,
+                               mocked_has_internet_connection):
+    mocked_has_internet_connection.return_value = True
+    mocked_valid_url.return_value = True
+
+    assert can_access_google_page("") == "Accessible", \
+        "Function must return 'Accessible' when " \
+        "internet connection and google url return True"
+
+
+def test_function_valid_google_url_is_correct(mocked_has_internet_connection):
+    mocked_has_internet_connection.return_value = True
+
+    assert can_access_google_page("http://www.google.com/") == "Accessible", \
+        "Function must return 'Accessible' if url is correct"
```

Comment:
You can remove this test as the function must return `True` if only both `mocked_valid_url` and `mocked_has_internet_connection` return True